interfaces/acados_matlab_octave/check_acados_requirements.m

```diff
@@ -36,7 +36,9 @@ function check_acados_requirements(varargin)
             error('Please set up CasADi yourself and try again.');
         end
         % download CasADi
-        CasADi_version = '3.5.5'; % TODO: update automatic download to later CasADi version.
+        CasADi_version = '3.6.6'; % TODO: update automatic download to later CasADi version.
+        CasADi_version_numeric = str2num(replace(CasADi_version, '.', ''));
```

Comment:
This is not really save, meaning that if double digit minor versions or patches are present this could be a nonunique mapping. But good enough for this, I don't have something to propose.

However, this comparison will not work if we want to link to a pre-release of casadi without a real version number.
Maybe add a boolean `later_than_36` which we then set manually?